MechJeb2/MechJebModuleStagingController.cs

```diff
@@ -94,23 +94,27 @@ public override void OnUpdate()
             if (HasStayingChutes(StageManager.CurrentStage - 1, vessel))
                 return;
 
-            //only fire decouplers to drop deactivated engines or tanks or fairings
-            bool firesDecoupler = InverseStageFiresDecoupler(StageManager.CurrentStage - 1, vessel);
-            if (firesDecoupler && !(InverseStageDecouplesDeactivatedEngineOrTank(StageManager.CurrentStage - 1, vessel) || HasFairing(StageManager.CurrentStage - 1, vessel)))
-                return;
-
-            //only decouple fairings if the dynamic pressure and altitude conditions are respected
-            if ((core.vesselState.dynamicPressure > fairingMaxDynamicPressure || core.vesselState.altitudeASL < fairingMinAltitude) &&
-                HasFairing(StageManager.CurrentStage - 1, vessel))
-                return;
+            //Always drop deactivated engines or tanks
+            if (!InverseStageDecouplesDeactivatedEngineOrTank(StageManager.CurrentStage - 1, vessel))
+            {
+                //only decouple fairings if the dynamic pressure and altitude conditions are respected
+                if ((core.vesselState.dynamicPressure > fairingMaxDynamicPressure || core.vesselState.altitudeASL < fairingMinAltitude) &&
+                    HasFairing(StageManager.CurrentStage - 1, vessel))
+                    return;
+
+                //only release launch clamps if we're at nearly full thrust
+                if (vesselState.thrustCurrent / vesselState.thrustAvailable < 1/1.2 &&

```

Comment:
Okay, I think I'm convinced SLT>1 is a fine check. But SLT of 1.05 at liftoff is still *really* low -- I think Saturn-V was always >1.15. I'm inclined to leave the delay because it still makes sense here -- gives a little extra spool up time which will reduce losses if pumps are still running -- and the default will probably work well for most rockets. For rockets where it doesn't, well, it's configurable. :-)